docs/pages/tutorials/reduce-build-times-with-cache.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Reduce build times with remote caching
+sidebar_label: Reduce build times
+---
+
+Devpod provides the ability to use a registry as a remote backend cache for your container builds. To enable it perform
+
+```
+./devpod-cli context set-options -o REGISTRY_CACHE={registry}
+```
+
+where registry follows the syntax "\{domain\}/\{project\}/\{repo\}", such as gcr.io/my-project/my-dev-env
+
+
+Ensure your docker environment has the image snapshotter enabled in your daemon (https://docs.docker.com/engine/storage/containerd/)
+```
+cat /etc/docker/daemon.json
+
+{
+  "features": {
+    "containerd-snapshotter": true
+  }
+}
+```
+
+Then prebuild your image to populate the cache
+
+```
+devpod build my-workspace
+```
+
+Now the next time you or anyone in your team needs to build an image, it can pull from the cache. If you've made changes to your environment, then devpod will detect these changes are 
+build on top of the existing image layers.
+
+#### How does devpod detect changes?
+
+Devpod parses your Dockerfile and .devcontainer.json to detect file paths that can affect your build context. Devpod traverses these files and makes a hash of the file contents to use as the 
+container's image tag. When you make changes to these files, the hash will change causing a rebuild. Thanks to the remote cache the previous layers will be built so only changes are needed. To 
+reduce build times it is recommened to follow docker's best practices when writing Dockerfiles.
+
+#### Why Kaniko?
+
+When DevPod uses the kubernetes driver and needs to build a devcontainer, but your local environment does not have a container runtime, it builds the container in the cluster. It does so using
+kaniko, kaniko builds the container in userspace and does not require super user priveledges. This is much more secure than docker in docker, where the docker daemon is either mounted locally on 
```

Comment:
```suggestion
kaniko. Kaniko builds the container in userspace and does not require super user priveleges. This is much more secure than docker in docker, where the docker daemon is either mounted locally on 
```

wouldn't mention that because by default we run workspace pods as root 😅